src/Resources/contao/classes/Ajax.php

```diff
@@ -332,7 +333,18 @@ public function executePostActions(DataContainer $dc)
 
 				// Set the new value
 				$varValue = \Input::post('value', true);
-				$strKey = ($this->strAction == 'reloadPagetree') ? 'pageTree' : 'fileTree';
+
+				switch ($this->strAction) {
+					case 'reloadPagetree':
+						$strKey = 'reloadPagetree';
+						break;
+					case 'reloadFiletree':
+						$strKey = 'fileTree';
+						break;
+					default:
+						$strKey = 'picker';
```

Comment:
this changes the fallback action, but I think that's fine if noone else has any concerns

Reply:
Well not really, only three actions are allowed here (see lines 268-270) and we cover each case. This `default` can be a bit misleading though...

Reply:
I know about the three actions. What I meant is that before this change, the default was `fileTree` if `$this->strAction != 'reloadPagetree'`. Now the default is `picker`. As I said, I'm fine with this, though I would assume we should not support an unknown action anyway.